deps/.editorconfig

```diff
@@ -0,0 +1,12 @@
+[*.cs]
+
+dotnet_diagnostic.CA1304.severity = none
```

Comment:
what are these getting disabled

Reply:
It is only for `deps` folder. To override default PowerToys diagnostics. I put it there because I couldn't build the solution because of some rules, as PowerAccent code is in this folder now. And I don't really want to change the entire solution for that :)